File: ProjectPO_Java/src/PO_java/LifeSymulator/Animals/Wolf.java
package PO_java.LifeSymulator.Animals;

import PO_java.LifeSymulator.Animal;
import PO_java.LifeSymulator.Tools.Cords;
import PO_java.LifeSymulator.World;

public class Wolf extends Animal {
    public Wolf(World world, Cords cords){
        strength = 9;
        initiative = 5;
        age = 0;
        sign = 'W';
        name = "Wolf";
        toKill = false;
        isDead = false;
        this.cords = cords;
        this.world = world;
    }
}

File: ProjectCppPO/Wolf.cpp
#include "Wolf.h"

Wolf::Wolf(World* world, Cords cords)
{
	features.strength = 9;
	features.initiative = 5;
	features.age = 0;
	features.look = 'W';
	features.name = "Wolf";
	features.toKill = false;
	location = cords;
	this->world = world;
}
